src/networkextension.cs

```diff
@@ -1293,7 +1293,11 @@ interface NEFilterSocketFlow {
 				
 	[iOS (9,0)][Mac (10,11, onlyOn64 : true)]
 	[BaseType (typeof (NETunnelNetworkSettings))]
+	[DisableDefaultCtor]
```

Comment:
As far as I can tell the default constructor is fine on this type + adding `[DisableDefaultCtor]` is a breaking change. You would need to add a non-generated constructor outside the binding files with `[Obsolete]` to preserve API compatibility.